test/let.wasm.fromBinary

```diff
@@ -10,15 +10,14 @@
   (drop
```

Comment:
Separately from this PR, since let was removed from the spec we could remove this test and the binary reading code for it.

Reply:
Ah, I actually did remove the other files for this test in this PR because they used RTTs and deleting them was easier than regenerating them. I missed this one, but I'll remove it, too.